demo/clusters/openshift/README.md

```diff
@@ -0,0 +1,147 @@
+      drop:
+        - ALL
+```
+
+## Using Multi-Instance GPU (MIG)
+
+Workloads that use the Multi-instance GPU (MIG) feature require MIG to be enabled on the worker nodes with [MIG-supported GPUs](https://docs.nvidia.com/datacenter/tesla/mig-user-guide/index.html#supported-gpus), e.g. A100.
+
+First, make sure to stop any custom pods that might be using the GPU. Disable the DCGM and DCGM Exporter of the NVIDIA GPU Operator by editing the operator's cluster policy (set `enabled: false`).
+
+Enable MIG via the MIG manager of the NVIDIA GPU Operator. **Do not configure MIG devices as the DRA driver will do it automatically on the fly.**
+
+1. In the GPU operator namespace, create a `ConfigMap` with the key `config.yaml` and the following content:
+
+  ```yaml
+  version: v1
+  mig-configs:
+    all-enabled:
+      - devices: all
+        mig-enabled: true
+        mig-devices: {}
+    all-disabled:
+     - devices: all
+       mig-enabled: false
+  ```
+
+2. Update the cluster policy to point the MIG manager to the new `ConfigMap`:
+
+  ```yaml
+  migManager:
+    config:
+      name: <configmap_name>
+  ```
+
+3. Label the target nodes with `nvidia.com/mig.config=all-enabled`:
+
+  ```console
+  $ oc label node <node> nvidia.com/mig.config=all-enabled --overwrite
+  ```
+
+MIG will be automatically enabled on the labeled nodes. For additional information, see [MIG Support in OpenShift Container Platform](https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/mig-ocp.html).
+
+You can verify the MIG status using the `nvidia-smi` command from a GPU driver pod:
+
+```console
+$ oc exec -ti nvidia-driver-daemonset-<suffix> -n nvidia-gpu-operator -- nvidia-smi
++-----------------------------------------------------------------------------------------+
+| NVIDIA-SMI 550.54.14              Driver Version: 550.54.14      CUDA Version: N/A      |
+|-----------------------------------------+------------------------+----------------------+
+| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
+|                                         |                        |               MIG M. |
+|=========================================+========================+======================|
+|   0  NVIDIA A100 80GB PCIe          On  |   00000000:17:00.0 Off |                   On |
+| N/A   35C    P0             45W /  300W |       0MiB /  81920MiB |     N/A      Default |
+|                                         |                        |              Enabled |
++-----------------------------------------+------------------------+----------------------+
+```
+
+If the MIG status is marked with an asterisk (i.e. `Enabled*`), it means that the setting could not be fully applied and you may need to reboot the node.
+This can happen on some cloud service providers (CSP) where the CSP blocks GPU reset for the GPUs passed into a VM.
```

Comment:
It might be worth mentioning that there is a WITH_REBOOT flag that can be set to do this automatically and point them to the first note at https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-operator-mig.html#enabling-mig-during-installation